SimTracker/TrackerMaterialAnalysis/test/trackingMaterialAnalyser_ForPhaseII.py

```diff
@@ -4,21 +4,11 @@
 
 process = cms.Process("MaterialAnalyser")
 
-readGeometryFromDB = False
-
-# Configuration and Conditions
-# We cannot read the geometry from the DB, since we have to inject out custom-made
-# material-budget grouping into the DDD of the detector. So we need to read the
-# geometry using the XMLIdealGeometryRecord.
-if readGeometryFromDB:
-# Global Tag and geometry via it
-  process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_condDBv2_cff")
-  from Configuration.AlCa.GlobalTag_condDBv2 import GlobalTag
-  process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:phase2_realistic', '')
-else:
-  process.load('Configuration.Geometry.GeometryExtended2026D41Reco_cff')
```

Comment:
on second thought, the phase-2 reco geometry will eventually come from DB as well. Can this change be reverted?

Reply:
See comment above